daml-lf/tests/daml-lf-test.sh

```diff
@@ -40,10 +39,6 @@ case "${MAIN##*.}" in
   dar)
     $DAML_LF_REPL testAll "$MAIN"
     ;;
-  daml)
```

Comment:
this script was never called with this argument (and anyway we remove it later in this PR)